webui/elm/Makefile

```diff
@@ -16,6 +16,9 @@ clean:
 	rm -f npm-debug.log
 	rm -f public/style.css
 	rm -f public/determined-ui.js
+
+.PHONY: clean-deps
+clean-deps:
 	rm -rf elm-stuff/ tests/elm-stuff/ node_modules/
```

Comment:
@hamidzr do you know if we still use this `elm-stuff` directory? I'm guessing `tests/elm-stuff` is not needed. I can't find any references of something that generates this directory.

Reply:
suggestion: cleaning the `clean-deps` :) to...
```
rm -rf node_modules/
```

Reply:
> @hamidzr do you know if we still use this `elm-stuff` directory? I'm guessing `tests/elm-stuff` is not needed. I can't find any references of something that generates this directory.

not the one for the under the tests directory but I think we want the main one